### PR TITLE
#655 Re-direct push/replace in list

### LINKED ISSUE #655: Bug: Clicking back on the Application list doesn't take you back to where you were

### PR DESCRIPTION
**Pull request by CarlosNZ**
Fixes #655 and #625 (Page titles).

As discussed with @andreievg

### CODE REVIEW

File: src/containers/Application/ApplicationCreate.tsx [hunk 1]
```diff
@@ -24,6 +25,8 @@ const ApplicationCreate: React.FC = () => {
     templateCode: type,
   })
 
+  usePageTitle(strings.PAGE_TITLE_NEW)
```
Comment (nmadruga) [inline src/containers/Application/ApplicationCreate.tsx:28]:
```suggestion
  usePageTitle(strings.PAGE_TITLE_CREATE)
```
I think it's more descriptive

File: src/utils/constants.ts [hunk 2]
```diff
@@ -117,4 +117,9 @@ export default {
   DATE_YEARS_AGO: 'years ago',
   FOOTER_TEXT: 'Powered by mSupply Application Manager',
   FOOTER_COPYRIGHT: '© 2020',
+  PAGE_TITLE_HOME: 'Dashboard | Application Manager',
+  PAGE_TITLE_LIST: 'Applications List |  Application Manager',
+  PAGE_TITLE_NEW: 'New Application | Application Manager',
```
Comment (nmadruga) [inline src/utils/constants.ts:122]:
```suggestion
  PAGE_TITLE_CREATE: 'New Application | Application Manager',
```

**Review by nmadruga — APPROVED** [review]
LGTM. I have left 1 suggestion - couldn't help myself : D

**Comment by CarlosNZ** [comment]
> LGTM. I have left 1 suggestion - couldn't help myself : D

Cool, thanks. Implemented suggestions with one click! :)

**Pull request merged**